scripts/release/mirror.test.js

```diff
@@ -76,7 +76,7 @@ describe('mirror', () => {
         ['13.1', '13.4'],
         ['14', '14'],
         ['14.1', '14.5'],
-        ['15', '15.1'],
+        ['15', '15'],
```

Comment:
…but this change seems correct.

Reply:
We'll have to fix it in some other way.

Reply:
https://github.com/mdn/browser-compat-data/pull/16594